tests/functional/harris/harris_2d.py

```
@@ -181,6 +184,9 @@ def plot(diag_dir, plot_dir):
             vmin=-2,
             vmax=2,
         )
+        ru.GetPressure(time).plot(
```

Ruff (0.11.9)

Undefined name `ru`

(F821)